plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2192,7 +2235,7 @@ public void registerCoreMembers() {
         // Kill - Used to detect when a player kills an NPC, player, entity, or player in a specific permission group.
         // Block - Used to detect when a player breaks, places, or collects blocks in the specified list.
         // Item - Used to detect when a player crafts, smelts, or fishes an item on the specified list.
-        // Itemdrop - TODO: Document Command Details
+        // Itemdrop - Used to detect when an item is dropped

```

Comment:
I would probably just delete this entire command's documentation as it's be deprecated for so long. Leave it registered for ancient script back-support.

Reply:
Rather for the moment, it should just be left untouched - and a separate PR / plan made regarding removing the entire listener setup